tests/test_bugbear.py

```diff
@@ -270,6 +270,8 @@ def test_b018_functions(self):
         expected = [B018(line, 4) for line in range(15, 25)]
         expected.append(B018(28, 4))
         expected.append(B018(31, 4))
+        expected.append(B018(32, 4))
+        expected.append(B018(33, 4))
```

Comment:
Why do we not need 34 here when we include 32 for modules tests? What am I missing here, cause the variable assign happens in a function it's fine?

Reply:
Oh it's a little confusing because the line numbers for the three different test cases are referring to the 3 different `b018_....py` files (but the line numbers are _almost_ the same for 2 of them. Added the same 3 lines to each of the files and corresponding test cases, in all cases the first 2 lines are bad and the 3rd is good

```python
    (1,)  # bad
    (2, 3)  # bad
    t = (4, 5)  # good
```